packages/react-native-client/android/src/main/java/org/membraneframework/reactnative/VideoRendererView.kt

```diff
@@ -1,26 +1,24 @@
 package org.membraneframework.reactnative
 
 import android.content.Context
+import com.fishjamcloud.client.media.VideoTrack
 import expo.modules.kotlin.AppContext
 import expo.modules.kotlin.views.ExpoView
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.launch
-import org.membraneframework.rtc.media.VideoTrack
-import org.membraneframework.rtc.ui.VideoTextureViewRenderer
 import org.webrtc.RendererCommon
 
 class VideoRendererView(
   context: Context,
   appContext: AppContext
 ) : ExpoView(context, appContext),
   RNFishjamClient.OnTrackUpdateListener {
-  var isInitialized = false
   var activeVideoTrack: VideoTrack? = null
   var trackId: String? = null
 
   private val videoView =
-    VideoTextureViewRenderer(context).also {
+    RNFishjamClient.fishjamClient!!.createVideoViewRenderer().also {
```

Comment:
Should we have some check here to ensure `fishjamClient` is not null? Or maybe use different types to avoid such code

Reply:
I added `lateinit` to `fishjamClient`. We're sure it's initialized, because it's initialized when the react-native-client module is created. And in order for the user to call anything, the module must be created. So `lateinit` sounds good here.